packages/webpack-plugin/src/plugin.ts

```diff
@@ -295,40 +295,38 @@ export class StylableWebpackPlugin {
         const options =
             loadStylableConfig(compiler.context, (config) => {
                 return isWebpackConfigProcessor(config)
-                    ? config.webpackPlugin(defaults, compiler)
+                    ? config.webpackPlugin(defaults, compiler, getTopLevelInputFilesystem(compiler))
```

Comment:
why do we need the fs passed here?

Reply:
To allow the webpack config to use the same FS as webpack itself. 

See the (non-inline) override test for an example.